templates/localgov-publication-page-header-block.html.twig

```diff
@@ -32,11 +32,19 @@
   </h1>
 
   {% if published_date %}
-    <div class="lgd-page-title-block__published"><span>Published: </span>{{ published_date }}</div>
-  {% endif %}
+  <div class="lgd-page-title-block__published">
+    {% trans %}
+      <span>Published:</span> {{ published_date }}
+    {% endtrans %}
+  </div>
+{% endif %}
 
-  {% if last_updated_date %}
-    <div class="lgd-page-title-block__updated"><span>Last updated: </span>{{ last_updated_date }}</div>
-  {% endif %}
+{% if last_updated_date %}
+  <div class="lgd-page-title-block__updated">
+    {% trans %}
+      <span>{{ 'Last updated: '|t }}</span>{{ last_updated_date }}
```

Comment:
You don't need the `|t` filter here, since it's inside a `{% trans %}` tag.

Reply:
Removed :-)